src/RestWrite.js

```diff
@@ -197,6 +200,37 @@ RestWrite.prototype.validateSchema = function () {
   );
 };
 
+// builds the default ACL depending on the className
+RestWrite.prototype.buildDefaultACL = function () {
+  if ((this.query && this.query.objectId) || this.data.ACL) {
+    return;
+  }
+  const defaultACL = {
+    currentUser: {
+      read: true,
+      write: true,
+    },
+  };
+  const aclOptions = Object.assign({}, this.config.defaultACL || defaultACL);
```

Comment:
```suggestion
  const aclOptions =  this.config.defaultACL || defaultACL;
```

Reply:
If I do that, tests fail and this.config.defaultACL gets mutated and then other tests fail.